setup.py

```diff
@@ -46,16 +43,13 @@
 contrib_require = [
     'editdistance>=0.3.1',
     'munkres<=1.0.12',
-    'Unidecode==0.4.19' if sys.version_info < (3, 6) else 'Unidecode>=0.4.19'
+    'Unidecode==0.4.19' if sys.version_info < (3, 11) else 'Unidecode>=0.4.19'
```

Comment:
this will not include 3.11, no?

Reply:
No, you can validate that here 

Python 3 build -> https://github.com/inveniosoftware-contrib/json-merger/actions/runs/14215509526/job/39831202902#step:4:49

Python 2 build -> 
https://github.com/inveniosoftware-contrib/json-merger/actions/runs/14215509514/job/39831202825?pr=82#step:5:63